lego/apps/events/views.py

```diff
@@ -193,6 +193,39 @@ def payment(self, request, *args, **kwargs):
         )
         return Response(data=payment_serializer.data, status=status.HTTP_202_ACCEPTED)
 
+    @decorators.action(
+        detail=False,
+        serializer_class=EventUserRegSerializer,
+        permission_classes=[permissions.IsAuthenticated],
+    )
+    def previous(self, request):
+        queryset = (
+            self.get_queryset()
+            .filter(
+                registrations__status=constants.SUCCESS_REGISTER,
```

Comment:
Is this filtering sufficient? Won't this return events where the user is in the waiting list as well? @odinuge

Reply:
Correcto.

This is the same as `upcoming`, and imo. it's nice to keep them similar. (@HoboKristian only dispalys  events with pool here: https://github.com/webkom/lego-webapp/pull/1541/files#diff-ac325e8453fd68444445e7f1267ead7cR372)

We can make a separate viewset, or add some custom filtering to `/api/v1/events/` to make it more generic.

Reply:
Ah, I see! 👍